src/components/ToDo/ToDo.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './todo.css';
+
+export function ToDo({ title, completed, user }) {
+  return (
+    <>
+      {title}
+      :
+      <p>
+        {' '}
+        {completed === true ? 'done' : 'not done'}
+        {' '}
+        by
+        {' '}
+        {user[0]?.name}
+        {' '}
+      </p>
+    </>
```

Comment:
let's use semantic tags, this part is confusing

Reply:
This is an <p> inside <li>, I didn't find the reason why it's not semantic.  But I tried to make inner jsx look nice inside <p>